src/core/net/tcp6.hpp

```diff
@@ -546,13 +628,43 @@ class Tcp : public InstanceLocator, private NonCopyable
      */
     Error HandleMessage(ot::Ip6::Header &aIp6Header, Message &aMessage, MessageInfo &aMessageInfo);
 
+    /**
+     * Automatically selects a local address and/or port for communication with the specified peer.
+     *
+     * @param[in] aPeer         The peer's address and port.
+     * @param[in,out] aToBind   The SockAddr into which to store the selected address and/or port.
+     * @param[in] aBindAddress  If true, the local address is selected; if not, the current address in @p aToBind is
+     * treated as a given.
+     * @param[in] aBindPort     If true, the local port is selected; if not, the current port in @p aToBind is treated
+     * as a given.
```

Comment:
nit: fix alignment

Reply:
Thanks. This is fixed in the latest update.